<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Календари и гороскопы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Год</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_year">
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_calendar">
      <property name="title">
       <string>Календарь</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="rb_eastern">
         <property name="text">
          <string>Восточный</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_tibetan">
         <property name="text">
          <string>Тибетский</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_zoroastrian">
         <property name="text">
          <string>Зароастрийский</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_is_date">
      <property name="text">
       <string>Ввод даты</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="gb_horoscope">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="title">
       <string>Гороскоп</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="rb_zodiac">
         <property name="text">
          <string>Зодиакальный</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rb_druid">
         <property name="text">
          <string>Друидов</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,0">
      <item>
       <widget class="QLabel" name="lb_day">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>День</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="sb_day">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>176</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>31</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_month">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Месяц</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_month">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Ваш магический гороскоп</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_calendar">
      <property name="text">
       <string>Введите год</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_horoscope">
      <property name="text">
       <string>Ввод даты отключён</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
